py/server/setup.py

```diff
@@ -53,7 +53,7 @@ def normalize_version(version):
     python_requires='>=3.7',
     install_requires=[
         'jpy>=0.11.0',
-        'deephaven-plugin',
+        'deephaven-plugin>=0.3.0.dev2',
```

Comment:
TODO: We should put a release version rather than referencing a dev version.

Reply:
Yep - I need to have the python side PR merged and released first, then can update this as appropriate.

Reply:
https://github.com/deephaven/deephaven-plugin/pull/7